fearless/Common/Network/Subquery/SubqueryRewardOperationFactory.swift

```diff
@@ -220,12 +271,57 @@ extension SubqueryRewardOperationFactory: SubqueryRewardOperationFactoryProtocol
         return operation
     }
 
+    func createDelegatorRewardsOperation(
+        address: String,
```

Comment:
use one code style for this file please. You used in 121 line "_ address", but here "address"